filebeat/input/journald/input.go

```diff
@@ -251,9 +261,17 @@ func (r *readerAdapter) Next() (reader.Message, error) {
 	content := []byte(data.Fields["MESSAGE"])
 	delete(data.Fields, "MESSAGE")
 
-	fields := make(map[string]interface{}, len(data.Fields))
-	for k, v := range data.Fields {
-		fields[k] = v
+	fields := r.converter.Convert(data.Fields)
+	fields.Put("event.kind", "event")
+	fields.Put("event.created", data.RealtimeTimestamp)
```

Comment:
[`event.created`](https://www.elastic.co/guide/en/ecs/current/ecs-event.html#field-event-created) is the time Filebeat generated this document/event (usually this means`time.Now().UTC()`). So I think this is incorrect (w/o knowing for sure what RealtimeTimestamp is).

`@timestamp` should the time from the journald event.

Reply:
Good catch!